src/password/scripts/directives.js

```diff
@@ -3,9 +3,33 @@
 
     var module = angular.module('user_management.password');
 
+    module.service('catchErrors', [
```

Comment:
I think this library is designed that each component can be used independently. This would mean I would have to define the the service in each place.